src/commands/rover/NicknameGroupCommand.js

```diff
@@ -0,0 +1,33 @@
+const Command = require('../Command')
+
+module.exports =
+class NicknameGroupCommand extends Command {
+  constructor (client) {
+    super(client, {
+      name: 'nicknamegroup',
+      properName: 'NicknameGroup',
+      aliases: ['rovernicknamegroup'],
+      description: "`<groupid>` Set the Group RoVer will use when setting users' nicknames.",
+
+      args: [
+        {
+          key: 'groupid',
+          label: 'groupid',
+          prompt: 'Nickname group',
+          type: 'string',
+          infinite: true
```

Comment:
This probably shouldn't be infinite, because this makes it return an array of strings instead of just one string. Even consider making this type: number. Not sure if Commando supports type: integer or int, but it does, use that even over number.